python/rascaline/rascaline/utils/clebsch_gordan/_equivariant_power_spectrum.py

```diff
@@ -0,0 +1,201 @@
+"""
+This module provides a convenience calculator for computing a single-center equivariant
+power spectrum.
+"""
+
+from typing import List, Optional, Union
+
+from ...calculators import SphericalExpansion
+from ...systems import IntoSystem
+from .._backend import Device, DType, Labels, TensorMap, TorchModule, operations
+from ._cg_product import ClebschGordanProduct
+from ._density_correlations import _filter_redundant_keys
+
+
+class EquivariantPowerSpectrum(TorchModule):
+    """
+    Computes an equivariant power spectrum descriptor, or a "lambda-SOAP".
+
+    For a full description of the hyper-parameters, see the corresponding
+    :ref:`documentation <soap-power-spectrum>`.
+    """
+
+    def __init__(
+        self,
+        cutoff,
+        max_radial,
+        max_angular,
+        atomic_gaussian_width,
+        center_atom_weight,
+        radial_basis,
+        cutoff_function,
+        radial_scaling=None,
+        *,
+        dtype: Optional[DType] = None,
+        device: Optional[Device] = None,
+    ):
```

Comment:
If we keep the `neighbors_to_properties` parameter, should it be set here instead of `compute/forward`? The point being to keep the compute function more similar from one calculator to another

Reply:
Yes makes sense :)